plugins/automation/spec/system/automation_edit_page_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+describe "DiscourseAutomation | Edit page", type: :system do
+  fab!(:admin)
+  fab!(:automation) { Fabricate(:automation, enabled: true) }
```

Comment:
fabricated automations are enabled by default!

```suggestion
  fab!(:automation)
```